script.js

```diff
@@ -0,0 +1,60 @@
+let startBtn = document.querySelector("#startButton");
+// this determines turn order
+let currentPlayer = 1;
+// this variable holds alternating player symbols
+let playerMarker;
+const tiles = document.querySelectorAll('.tile');
+let gameState = ["", "", "", "", "", "", "", "", ""];
+//this is a boolean to regulate whether a player can click a tile. 
+let notClicked;
+
+function incrementCurrentPlayer() {
+    currentPlayer++;
+    return currentPlayer;
+}
+
+function decideTurn(currentPlayer, playerMarker) {
+    if (currentPlayer % 2 == 0) {
+        playerMarker = 'o'
+    } else {
+        playerMarker = 'x'
+    }
+    return playerMarker;
+}
+
+function addToScreen(tile, playerMarker) {
+    tile.innerHTML = playerMarker;
+}
+
+function alterGameState(gameState, clicked, playerMarker) {
+    if(gameState[clicked] == "") {
+        gameState[clicked] = playerMarker;
+        console.log(gameState[clicked]);
+        console.log(gameState);
+        return true;
+    } else {
+        return false;
+    }
+}
+
+startBtn.addEventListener("click", () => {
+    startBtn.style.opacity = 0.5;
+    tiles.forEach(function (tile) {
+        tile.addEventListener('click', () => {
```

Comment:
Does this get removed? i.e. If I press start game again will there be 2 event listeners on this square

Reply:
this is fixed